values.yaml

```diff
@@ -53,3 +53,49 @@ reconcile:
   overwriteZk: true
   successfulJobsHistoryLimit: null
   failedJobsHistoryLimit: null
+
+service:
+  type: ClusterIP
+  port: 9000
+
+ingress:
+  ## Set to true to enable ingress record generation
+  ##
+  enabled: true
+
+  ## When the ingress is enabled, a host pointing to this will be created
+  ##
+  host: cmak.local
+
+  ## The Path to CMAK. You may need to set this to '/*' in order to use this
+  ## with ALB ingress controllers.
+  ##
+  path: /
+
+  ## Ingress annotations done as key:value pairs
+  ## For a full list of possible ingress annotations, please see
+  ## ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md
+  ##
+  annotations: {kubernetes.io/ingress.class: nginx}
+
+  ## Enable TLS configuration for the hostname defined at ingress.hostname parameter
+  ## TLS certificates will be retrieved from a TLS secret with name: {{- printf "%s-tls" .Values.ingress.hostname }}
+  ## You can use the ingress.secrets parameter to create this TLS secret or relay on cert-manager to create it
+  ##
+  tls: false
+
+  ## If you're providing your own certificates, please use this to add the certificates as secrets
+  ## key and certificate should start with -----BEGIN CERTIFICATE----- or
```

Comment:
This explanation looks like a copy paste from somewhere, and it is referring to non existant README.md.
It will be enough to just keep a first line as a description.